src/scripts/containers/UploadVideoStatusContainer.js

```diff
@@ -4,11 +4,10 @@ import React, { Component } from 'react'
 import VideoStatus from 'components/VideoStatus'
 import type { RootState } from 'types/ApplicationTypes'
 import VideoRecord from 'records/VideoRecords'
-import UploadRecord from 'records/UploadRecords'
 
 type Props = {
-  video: VideoRecord,
-  state: UploadRecord
+  video: VideoRecord, // TODO: seems a bit too much
+  state: VideoRecord
```

Comment:
We definitely should have a `prop` called `state`, that is bound to confuse the hell out of people :P. What does this actually mean?